updates/stable.json

```diff
@@ -1,31 +1,23 @@
 {
   "stream": "stable",
   "metadata": {
-    "last-modified": "2020-08-12T12:12:16Z"
+    "last-modified": "2020-08-24T22:03:38Z"
```

Comment:
I usually just set this to the same value that got generated in the other file since that value was generated with a tool. So `2020-08-24T22:19:06Z`, in this case. Might be easier for you to do that in the future.